examples/producer-consumer/product.pony

```diff
@@ -0,0 +1,7 @@
+struct val Product
```

Comment:
Because is something without behaviour and with a state that can't even be changed.